brdnet/run_plier.R

```diff
@@ -0,0 +1,47 @@
+# This script runs PLIER on the data from download_categorized_data.ipynb
+
+# CRAN dependencies
+library('dplyr')
+
+# Bioconductor dependencies
+library('biomaRt')
+
+# Github dependencies
+library('PLIER')
+
+data("canonicalPathways")
+data("bloodCellMarkersIRISDMAP")
+data("svmMarkers")
+
+data.dir <- '../data'
+
+# Set the random seed
+set.seed(42)
+
+# Load and combine data from download_categorized_data.ipynb
+disease.df <- read.table(file.path(data.dir, 'plier_disease.tsv'), sep = '\t', stringsAsFactors = FALSE, header = TRUE)
+healthy.df <- read.table(file.path(data.dir, 'plier_healthy.tsv'), sep = '\t', stringsAsFactors = FALSE, header = TRUE)
+
+# Combine the disease and healthy dataframes
+all.df <- merge.data.frame(healthy.df, disease.df, by='row.names')
+
+row.names(all.df) <- all.df['Row.names'][[1]]
+all.df['Row.names'] <- NULL
+
+allPaths <- combinePaths(bloodCellMarkersIRISDMAP, canonicalPathways, svmMarkers)
```

Comment:
Since it seems like a lot of your computational heavy lifting is done in this script, it would be helpful to include more info on this combinePaths function, even by saying package::combinePaths.

Reply:
Good call, I added a comment above the line and explicitly called the function from PLIER.